core/src/com/unciv/logic/automation/Automation.kt

```diff
@@ -352,10 +352,12 @@ object Automation {
         val stats = tile.getTileStats(null, civInfo)
         var rank = rankStatsValue(stats, civInfo)
         if (tile.improvement == null) rank += 0.5f // improvement potential!
+        if (tile.isPillaged()) rank += 0.6f
```

Comment:
Remember that this might be a pillaged road. Pillaged roads should not be adding stats

Reply:
Why not? Workers should prioritize pillaged roads as well as improvements

Reply:
Oh, because this destiny be caught in automate connections...
Good catch! 👍